GameDev/week6/livestream/outline.md

```diff
@@ -0,0 +1,30 @@
+Outline:
+
+Hey all, today we’ll be looking at how to put sounds in your Unity projects. I’ll be going over how to play sounds through code, how to adjust those sounds in Unity, and how to find and best use free sound assets online.
+
+Now, hopefully you’ve already downloaded and opened the same project I have in front of me, which you can find and download at this github link: https://github.com/chris8736/bit_temp_mds/tree/main 
+
+Audio Source basics (5 mins)
+	-create an Audio Source component with clip
+	-play with values during runtime
+Component Referencing (5 mins)
```

Comment:
These section headers would benefit from being bolded or changed to a header https://www.markdownguide.org/cheat-sheet/